Userland/Utilities/watch.cpp

```diff
@@ -188,29 +173,28 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     } else {
         TRY(Core::System::pledge("stdio proc exec"));
 
-        struct timeval interval;
+        Duration interval;
         if (opt_interval <= 0) {
-            interval = { 0, 100000 };
+            interval = Duration::from_milliseconds(100);
         } else {
-            interval = { opt_interval, 0 };
+            interval = Duration::from_seconds(opt_interval);
         }
 
-        auto now = get_current_time();
+        auto now = UnixDateTime::now();
```

Comment:
I don't think we actually need the Unix time here, can we use `MonotonicTime` instead?

Reply:
I didn't know that was a thing. That makes more sense here. I've changed it :^)